x-pack/plugins/monitoring/public/components/chart/get_color.js

```diff
@@ -17,6 +17,23 @@ export function getColor(app, index) {
   let seriesColors;
   if (app === 'elasticsearch') {
     seriesColors = ['#3ebeb0', '#3b73ac', '#f08656', '#6c478f'];
+  } else if (app === 'apm') {
+    // From https://github.com/elastic/elastic-charts/blob/master/src/utils/themes/colors.ts
+    seriesColors = [
```

Comment:
Don't we have an eui palete for this?

Reply:
Do you know of an example of using the color palettes within Kibana?

Reply:
There's a palette library that provides a default set of colors to be used. https://elastic.github.io/eui/#/utilities/color-palettes

Reply:
Got it, thanks!